packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -226,7 +227,7 @@ export const FilterButton = forwardRef(function FilterButton<
             key={option.value}
             option={option}
             onChange={handleChange}
-            activeValue={value || activeValue}
+            activeValue={activeValue}
```

Comment:
nice

Reply:
i feel like there was a reason that we had this the way that it was, but I can't remember why. feel free to ignore if this can been tested